mmv1/templates/terraform/examples/cloud_run_service_pubsub.tf.erb

```diff
@@ -0,0 +1,62 @@
+# [START cloud_run_service_pubsub_service]
+resource "<%= ctx[:vars]['primary_resource_type'] %>" "<%= ctx[:primary_resource_id] %>" {
+    name     = "<%= ctx[:vars]['cloud_run_service_name'] %>"
+    location = "us-central1"
+    template {
+      spec {
+            containers {
+                image = "gcr.io/cloudrun/hello"
+            }
+      }
+    }
+    traffic {
+      percent         = 100
+      latest_revision = true
+    }
+}
+# [END cloud_run_service_pubsub_service]
+
+# [START cloud_run_service_pubsub_sa]
+resource "google_service_account" "sa" {
+  account_id   = "cloud-run-pubsub-invoker"
+  display_name = "Cloud Run Pub/Sub Invoker"
+}
+# [END cloud_run_service_pubsub_sa]
+
+# [START cloud_run_service_pubsub_run_invoke_permissions]
+resource "google_cloud_run_service_iam_binding" "binding" {
+  location = google_cloud_run_service["<%= ctx[:primary_resource_id] %>"].location
+  service = google_cloud_run_service["<%= ctx[:primary_resource_id] %>"].name
+  role = "roles/run.invoker"
+  members = ["serviceAccount:<%= ctx[:vars]['pubsub_sa'] %>"]
```

Comment:
Should this be reusing the service account created above? If so this (and all other uses of this var) could reference that instead of requiring a separate variable:

```suggestion
  members = ["serviceAccount:${google_service_account.sa.email}"]
```